config/database.yml

```diff
@@ -5,21 +5,24 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
```

Comment:
> Hopefully this doesn't interrupt anyone's local dev environments too much currently

What about ignoring `database.yml` and adding a `database.example.yml` or creating the postgresql options in another key instead of `default`?